package.json

```diff
@@ -41,13 +41,14 @@
     "grunt-contrib-jshint": "^1.0.0",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-env": "latest",
+    "grunt-if": "https://github.com/binarymist/grunt-if/tarball/master",
     "grunt-jsbeautifier": "^0.2.12",
     "grunt-mocha-test": "^0.12.7",
     "grunt-nodemon": "^0.4.2",
-    "grunt-if": "https://github.com/binarymist/grunt-if/tarball/master",
     "grunt-npm-install": "^0.3.0",
     "grunt-retire": "^0.3.12",
     "mocha": "^2.4.5",
+    "opn": "^4.0.2",
```

Comment:
Should this be moved to dependencies section as the code (server.js) has a direct  dependency on this module?